src/tests/Loader/classloader/MethodImpl/InternalMethodImplTest.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- Needed for mechanical merging of all remaining tests, this particular project may not actually need process isolation -->
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
+  </PropertyGroup>
```

Comment:
```suggestion
```

Reply:
I did keep it so I can execute tests on coreclr only

https://github.com/dotnet/runtime/blob/6771302b5ebced11335edb8fdecadf61ff9100ae/src/tests/JIT/Directed/pinvoke/jump.ilproj#L3-L4